### ISSUE TITLE
rimraf not deletes hidden files like .htaccess

### ISSUE THREAD

**Issue by naiduprakash** [isaacs/rimraf#167]
**rimraf ./public/***  deletes all folders and files except the  **.htaccess** file in windows system

**Comment by Exocomp** [comment]
I ran into the same issue with another hidden file, would be nice to include hidden file support.

**Comment by isaacs** [comment]
The `*` only matches files that are not prefixed with a `.`.  Use `rimraf ./public` or `rimraf ./public/{*,.*}` instead.

**Comment by SrBrahma** [comment]
> The `*` only matches files that are not prefixed with a `.`. Use `rimraf ./public` or `rimraf ./public/{*,.*}` instead.

Sorry for commenting on such a old post, but can you explain how it works?

**Comment by DesignByOnyx** [comment]
the `{*,.*}` pattern is a glob pattern. If you aren't familiar with glob patterns, it's worth a deep dive into google. In layman's terms, the curly braces indicate "here's a special glob pattern". Inside the curly braces are two separate patterns (separated by a comma), the first matches everything `*`, but this ignores hidden files by default. The second pattern explicitly matches hidden files `.*`.

**Comment by SrBrahma** [comment]
@DesignByOnyx many thanks!
